### ISSUE TITLE
macs2 command not found

### ISSUE THREAD

**Issue by priyasshah** [taoliu/MACS#95]
I have Python 2.7.10 on a Mac. 

I believe there is something wrong with the way I am calling macs2. I always get this error:

otp588416ots:~ Priya$ macs2 callpeak -t ChIP.bam -c Control.bam -f BAM -g hs -n test -B -q 0.01
-bash: macs2: command not found

After installing macs2, my computer says the file is located in several folders (maybe from multiple installation attempts):

otp588416ots:~ Priya$ sudo find / -name macs2
Password:
find: /dev/fd/3: Not a directory
find: /dev/fd/4: Not a directory
/Library/Python/2.7/site-packages/MACS2-2.1.0.20150731/bin/macs2
/Library/Python/2.7/site-packages/MACS2-2.1.0.20150731/build/scripts-2.7/macs2
/opt/local/Library/Frameworks/Python.framework/Versions/2.7/bin/macs2
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/MACS2-2.1.0.20150731/bin/macs2
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/MACS2-2.1.0.20150731/build/scripts-2.7/macs2

Setting the PYTHONPATH to any of these directories does not change the error I receive when calling macs2. Are the multiple locations confusing my computer? Any help in resolving this problem would be greatly appreciated! Thank you!

**Comment by taoliu** [comment]
How did you install MACS2? Through pip or through setup.py script? Also, please post the message at the end of installation. PYTHONPATH has nothing to do with executable commands. The executable macs2 script should be found in PATH — echo $PATH to check your PATH variable.

Tao

**Comment by priyasshah** [comment]
Thanks for your response, Tao. I originally tried through pip install, and then through setup.py (again, maybe multiple installs causing a problem?). I didn't save the output history in bash, so I'm afraid I can't tell you what the message at end of installation was. I will say that there were a couple of attempts in which I didn't have permissions, so I went back and used sudo chmod to give myself permissions to the appropriate folders. 

I originally assumed that I needed to set the PYTHONPATH because MACS2 is a PYTHON package. Based on your comment that the PYTHONPATH is irrelevant, I added this PATH to my .bash_profile:
export PATH=/opt/local/Library/Frameworks/Python.framework/Versions/2.7/bin:$PATH

Now the macs2 command works fine!
